src/app/clusters/scenes-server/scenes-server.cpp

```diff
@@ -655,11 +892,20 @@ void ScenesServer::HandleRemoveScene(HandlerContext & ctx, const Commands::Remov
     ReturnOnFailure(AddResponseOnError(
         ctx, response, sceneTable->RemoveSceneTableEntry(ctx.mCommandHandler.GetAccessingFabricIndex(), scene.mStorageId)));
 
-    // Update Attributes
-    MatterReportingAttributeChangeCallback(ctx.mRequestPath.mEndpointId, Id, Attributes::SceneCount::Id);
-    MatterReportingAttributeChangeCallback(ctx.mRequestPath.mEndpointId, Id, Attributes::RemainingCapacity::Id);
+    // Update SceneInfoStruct Attributes
+    Structs::SceneInfoStruct::Type * sceneInfo =
+        GetSceneInfoStruct(ctx.mRequestPath.mEndpointId, ctx.mCommandHandler.GetAccessingFabricIndex());
+    Optional<bool> sceneValid;
+    if (nullptr != sceneInfo)
+    {
+        sceneValid = (req.groupID == sceneInfo->currentGroup && req.sceneID == sceneInfo->currentScene) ? Optional<bool>(false)
+                                                                                                        : Optional<bool>();
+    }
 
-    ReturnOnFailure(UpdateLastConfiguredBy(ctx, response));
```

Comment:
Why do we no longer need UpdateLastConfiguredBy here?  And if we do, why did CI not catch this?  Please add tests.

Reply:
We do need it, a test will be added.

Reply:
Test added